BMM.Core/Translation/en/main.json

```diff
@@ -290,7 +290,9 @@
     "AppInfoDeviceVersion": "Device version: ",
     "LoggedInAs": "Signed in as",
     "LogoutConfirm": "Are you sure you want to log out?",
-    "Logout": "Log out"
+    "Logout": "Log out",
+    "UserVoiceHeader": "User Voice",
+    "UserVoiceText": "Let us know about your idea on how we can improve BMM."
```

Comment:
Can you change this please?
"Got ideas on how to improve BMM? Tell us about it."